tests/test_visitors/test_ast/test_classes/test_parent_class_visitor.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.consistency import (
+    WrongParentClassListDef,
+)
+from wemake_python_styleguide.visitors.ast.classes import (
+    WrongParentClassListVisitor,
+)
+
+
+@pytest.mark.parametrize('code', [
+    'class TestClassName(FirstClassName, SecondClassName, object): ...',
```

Comment:
Can you please move this example into the global variable?

Like: `extra_object = 'class TestClassName(FirstClassName, SecondClassName, object): ...'`
This way it would be more readable.

Reply:
The same applies to all other examples.